tools/cldr-apps/src/main/webapp/js/new/cldrOtherSpecial.js

```diff
@@ -0,0 +1,146 @@
+// "use strict";
+// TODO: modernize, make strict, possibly a class
```

Comment:
'…possibly a class' - +1 on that, is that being tracked in any of the issues?

Reply:
I'll make sure this TODO is dealt with somehow before this ticket is closed; maybe a new ticket. So far, cldrOtherSpecial.js is just a placeholder. I'm re-implementing what were "specials", such as `#forum`, one-by-one, and also adding some new "specials" like `#about` and `#admin`, but without using the js/specials folder or dojo. I'm still studying how to capture what the "specials" all share in common in a more object-oriented way.